Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [workspace]
-members = ["crates/*", "examples/*"]
-exclude = ["examples/streams-ts/"]
+members = ["crates/*", "examples/*", "crates/parsers_codama/yellowstone-vixen-meteora-parser", "crates/parsers_codama/yellowstone-vixen-pump-fun-parser"]
```

Comment:
Just have this be flat then with `crates/meteora-parser` but have the create name be `yellowstone-vixen-pumpfun-parser`